src/classes/MAPR_PrimaryAffiliations_TEST.cls

```diff
@@ -430,15 +430,9 @@ private class MAPR_PrimaryAffiliations_TEST {
 
         List<Affl_Mappings__c> primaryAfflMappings = UTIL_CustomSettingsFacade.getPrimaryAfflMappings();
 
-        try{
-            Test.startTest();
-                MODL_PrimaryAfflMapper primaryAfflModel = maprPrimaryAfflInstance.getModelFromMapping(primaryAfflMappings[0]);
-            Test.stopTest();
-            System.assert(false, 'Always throw an exception when Primary Affl Field is not valid and when Affiliation Record Type is Enforced ');
-        } catch (ERR_ExceptionHandler.AffAccountRecordTypeMappingsException ex) {
-            System.assert(ex.getMessage().contains(Label.afflAccoutMappingError));
-        }   
-
+        Test.startTest();
+        System.assertEquals(null, maprPrimaryAfflInstance.getModelFromMapping(primaryAfflMappings[0]), 'Affiliation mapping with invalid Contact field should return null value.');
```

Comment:
Please make an assignment using the method and move the assert outside of the test block. I've been lax on this, but we want to try and be certain only the executing code we are testing is inside the test block since running metrics on its limits may be helpful in the future.